http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/GKEInterpreter.scala

```diff
@@ -0,0 +1,390 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package monitor
```

Comment:
I'm not certain this class belongs in `monitor` -- we have the Dataproc and GCE interpreters in the `util` package

Reply:
I originally had it there, but I moved it because I wanted to unit test some private methods. 

It's a bit annoying because we test these util interps as part of the LeoPubsubMessageSubscriber, which is in monitor. I'll split those tests out into a separate file and move this to util.

Reply:
Can probably annotate methods as `private[leonardo]` to make available to tests